cell_feature_data/bin/create_dataset.py

```diff
@@ -51,7 +51,7 @@ def create_single_dataset(output_path: str, for_megaset: bool = False):
     file_path = questionary.path(
         "Enter a valid file path:",
         validate=lambda text: True if len(text) > 0 else "File path cannot be empty.",
-    ).ask()
```

Comment:
Why did you need to swap for `unsafe_ask`? Do we just want to ignore keyboard interrupt commands (e.g. Ctrl + C)

Reply:
By using unsafe_ask, the keyboard interrupts(Ctrl+C) won't be caught by each individual question. Instead, we can catch and handle these interrupts in one place as exceptions. I have tested each unsafe_ask and they worked as expected.

More details in Questionary [documentation ](https://github.com/allen-cell-animated/cell-feature-data/pull/110)